_posts/2016-04-06-service_objects.md

```diff
@@ -0,0 +1,65 @@
+---
+layout: post
+title: Service Objects
+category: Ruby
+author: dmp0x7c5
+---
+
+Imagine you have really complex legacy code to work with and you run git blame just to append more and more names to your personal blacklist... Service objects will certainly help you. If you haven't heard about them before, here comes the shortest possible definition: Service object is just a plain old ruby object (PORO) responsible for ONE thing. Some people call them just object. That's it. It doesn't matter how you implement it or where you place it, the ONE thing is the key.
```

Comment:
like the beginning 👍 
when I first time read about Service Objects, I thought there are some gem, or something inheriting from rails lib - not a PORO ones